File: ua.js
const fs = require('fs');
const faker = require('faker');

const saveFile = process.argv[2];
const amount = parseInt(process.argv[3]);

if (!saveFile || isNaN(amount)) {
    console.log('Usage: node ua.js [save file] [amount]');
    process.exit(1);
}

const userAgents = [];

for (let i = 0; i < amount; i++) {
    userAgents.push(faker.internet.userAgent());
}

fs.writeFileSync(saveFile, userAgents.join('\n'));

console.log(`${amount} user agents saved to ${saveFile}`);

File: ua.py
import sys
import faker

def generate_user_agents(save_file, amount):
    fake = faker.Faker()
    user_agents = [fake.user_agent() for _ in range(amount)]
    
    with open(save_file, 'w') as f:
        f.write('\n'.join(user_agents))
    
    print(f"{amount} user agents saved to {save_file}")

if __name__ == "__main__":
    if len(sys.argv) != 3:
        print("Usage: python ua.py [save file] [amount]")
        sys.exit(1)

    save_file = sys.argv[1]
    amount = int(sys.argv[2])

    generate_user_agents(save_file, amount)
